src/kakarot/kakarot.cairo

```diff
@@ -127,26 +127,6 @@ func execute_at_address{
     );
 }
 
-// @notice Set the account registry used by kakarot
```

Comment:
see my comments regarding the `Deployer`, we may need to adapt to using the `Deployer` here